# **Part 4. Phylogeny tree of D.muelleri repeats**

Laboratory journal: [Repeats_in_DM](https://colab.research.google.com/drive/1roHFfcjmrdVqhQhsiRW4li5k_tSIXpoZ?usp=sharing)

## **Introduction**

In this section, the focus was placed exclusively on repeats from D. muelleri. The study followed a comprehensive plan beginning with the extraction and preparation of "Unknown" repeats, followed by filtering out short sequences and simple sequence repeats (SSR).

Next, the analysis involved clustering using CD-HIT and estimating the coverage of each cluster. Prior to alignment, only clusters contributing more than 0.1% to the genome were selected. The selected clusters were then aligned using MAFFT, and a phylogenetic tree was constructed with IQ-TREE.

Ultimately, an interesting cluster was identified that accounts for 15% of the genome of this microsporidium. The consensus sequence generated by CD-HIT for this cluster was further analyzed for the presence of open reading frames (ORFs), revealing eight ORFs. However, these ORFs are small and do not resemble genuine ones, indicating that further investigation is needed.
 
## **Methods**

**Plan**:

- Extracting and preparing "Unknown" repeats
- Filtering of short sequences and simple repeats (filter those have >30% simple repeats based on TRF output)
```
seqkit seq -g --min-len 200 unknown_repeats.fa > unknown_200.fa
trf unknown_200.fa 2 5 7 80 10 50 1000 -l 10 -d -h -m > trf_dm_solo.txt
```
- Clusterization (CD-HIT)
```
 cd-hit-est -i filtered.fa \
           -o filtered_cdhit \
           -c 0.80 \
           -n 4 \
           -d 0 \
           -M 16000 \
           -T 2
```
- Cluster coverage estimation
- Alignment (MAFFT)
```
mafft --auto dm_outgroup2.fa > aligned_dm_outgroup.fasta
```
- Building a tree with IQ-TREE (LINE/CR1 as an outgroup)
```
iqtree -s aligned_dm_outgroup.fasta -m TEST -bb 1000 -nt AUTO -o LINECR1_911,LINECR1_1139,LINECR1_1045
```
- Selection of "interesting" families and a BLAST test

## **Results**

**The phylogenetic tree**

A phylogenetic tree was constructed for 87 clusters of repeats, each contributing more than 0.1% to the genome, using repeats of the LINE/CR1 family—also found in this organism—as an outgroup. The trees are presented in both a standard and a circular layout, and additional data indicating the percentage of the genome occupied by each cluster were overlaid. 

*Figure 1. Visualization of phylogeny tree*
<img width="759" alt="Снимок экрана 2025-02-17 в 11 21 43" src="https://github.com/user-attachments/assets/7380d27e-d520-4c5e-9570-7fcbd7dffc8e" />

*Figure 2. Visualization of phylogeny tree (circle variant)*
<img width="829" alt="Снимок экрана 2025-02-17 в 11 22 05" src="https://github.com/user-attachments/assets/e8c96d26-1eec-4316-a06e-e2de07df0652" />

However, the alignment visualization still does not appear optimal, likely due to the use of different types of repeats in a single analysis.

*Figure 3. Visualization of alignment*
<img width="1366" alt="Снимок экрана 2025-02-16 в 17 33 37" src="https://github.com/user-attachments/assets/699a456a-edd2-4deb-8bba-b1c246df3205" />

Two clusters clearly stand out on the tree: cluster 92, which occupies over 15% of the D. muelleri genome, and cluster 7, which occupies more than 4%. Their substantial genomic contributions warrant a more detailed examination. 

---
**Cluster 92 (occupies >15% of the genome)**

Let us first consider cluster 92, which consists of 9 sequences. The alignment of sequences within this cluster is shown below. 

*Figure 4. Visualization of alignment for cluster 92*
<img width="1366" alt="Снимок экрана 2025-02-17 в 20 07 02" src="https://github.com/user-attachments/assets/1c56c67e-d879-43a5-ab1b-d93ff1088754" />

An ORF search in the consensus sequence for this cluster, identified 8 ORFs. However, these ORFs are relatively short and may not be genuine. 

*Figure 5. ORFs that was found in rnd-1_family-3#Unknown*
<img width="1220" alt="Снимок экрана 2025-02-17 в 20 47 14" src="https://github.com/user-attachments/assets/9503b0fa-dc89-47dc-8a8a-cf21cacb9de4" />

The longest one, when subjected to a BLAST search, yielded the following results.

*Table 1. BLAST results for longest ORF*
| Select for downloading or viewing reports | Description                                                                                           | Scientific Name                             | Max Score | Total Score | Query Cover | E value | Per. Ident | Acc. Len | Accession  |
|---------------------------------------------|-------------------------------------------------------------------------------------------------------|---------------------------------------------|-----------|-------------|-------------|---------|------------|----------|------------|
| Select seq sp\|Q6R7G5.1\|                   | RecName: Full=Uncharacterized protein ORF60 [Ostreid herpesvirus 1 (isolate France)]                    | Ostreid herpesvirus 1 (isolate France)        | 26.6      | 26.6        | 35%         | 8.9     | 36.84%     | 411      | Q6R7G5.1   |
| Select seq sp\|A4IG42.1\|                   | RecName: Full=TRPM8 channel-associated factor homolog [Danio rerio]                                     | Danio rerio                                 | 26.6      | 26.6        | 84%         | 9.8     | 34.78%     | 912      | A4IG42.1   |

Consesus sequence for cluster 92:
```
>rnd-1_family-3#Unknown ( RepeatScout Family Size = 1464, Final Multiple Alignment Size = 100, Localized to 9327 out of 11027 contigs )
ATATATATTACATGTTGACAACATCCATAGCTGCCAACCCGGAGAGTTGTCTCCTACCGT
GGTCTGTGACCTTTGGTTCTTGGTGTTCCAATAATACGTCACTTCCTAAGCTACCCAGAA
GCCCAGCTGTATACGCGATTTTCGTTTTATCAATGACTCAGCAGGATACAGGTAATTAGT
GGCTTTTCATGAATTTTTATTTTTCACTACTTTTTGCCGAGATGCGTCGCCGCACTCATC
CCCACACCTAGGAGAGTCCCTTCTGCTTCATTGTCATATTAGACCAAGTCTTAATCAGCT
ATACCACACTGGAACAACTGTATATTATACTATAATTATATAGTTNTGACCGCTTAATGT
CATTCCCTCATTATTTATTAAAGAGTGACGTCAGCGGGATTCGAACCACGATACTTAAAT
TTTACATTCAAGAGTGTTAACCACTGAGCTATCTGGCCCCTTAAATATTGTTTAAATCTA
TAATACTCTATAATAAATGTGTATAAGCTTGAAAGAAATGGGTCTATAACTTTGGAAAAA
TAGGTTTATAACTATGAAAAGCCCGTATAAAAGACGACTTGGTCTAATTGAAAATAGAAC
AGAATAGCTCTTTGAGAGTGGACATAAGACGGCGACACATTAAGCAAAACATAGTGAA
```
If the consensus sequence for this cluster occupies approximately 2% of the genome, then one of the sequences, presented individually, occupies over 8%, which is also noteworthy.

```
>rnd-1_family-0#Unknown ( RepeatScout Family Size = 7196, Final Multiple Alignment Size = 100, Localized to 9327 out of 11027 contigs )
TATATATATTATATATATTACATGCTGACAACATCCATAGCTGCCAACCCGGAGAGTTGT
CTCCNACCGTGGTCTTGTGACCTTTGGTTCCTGGTGTTCCAATAATACGTCACTTCCTAA
GCTACCCAGAAGCCCAGCTGTATACGCGATTTTCGTTTTATCAATGACTCGGCAGGATAC
AGGTAATTAGTGGCTTTTCATGAATTTTTATTTTTCACTACTTTTTGCCGAGATGCGTCG
CCGCACTCATCCCCACACCTAGGAGAGTCCCTTCTGCTTCATTGTCATATTAGACCAAGT
CTTAATCAGCTATACCACACTGGAACAACTGTATATTAATACTATATTTCTATAGTTTTT
GACCGCTTAATGTCATTCCCTCATTATTTATTAAAGAGTGACGTCAGCGGGATTCGAACC
ACGATACTTAAATTTTACATTCAAGAGTGTTAACCACTGAGCTATCTGGC
```
---

**Cluster 7 (occupies >4% of the genome)**

Now, let us consider cluster 7, which consists of a single sequence in which 7 ORFs were identified. These ORFs are also small and therefore may be unreliable, warranting further investigation to confirm their functionality.

```
>rnd-1_family-6#Unknown ( RepeatScout Family Size = 1272, Final Multiple Alignment Size = 100, Localized to 9327 out of 11027 contigs )
TATACAGTGAATCCCCTTAAATATGAACACTCTTAAATATGAACACTCCTCTTTTTTAAA
ATTTTATAATAAAATCATTAATAAAAATGACCCTTACCCTTGAACGTGAACAGAAAATAA
AAAAATTTGTTAATAAAAACCATTATATTTTTTTATTTCAAAAAAATTGACCCTTAAATA
TGAACAAAGAAAAATTTAAAAATAATAAATACAATAAATAAAAAAAGTGTGAAATAATAA
AAAGGATTGGTTCTCTATCGGTTTAGCCACCCCCACCTTTTAAATAAAANTGCTACACGG
AAATCTGGATCCGGAAACTCGGCCAAAAAAATATTTCTAAATATTTGGGCCAGATAGCTC
AGTTGTTAACACCATTAGANGAAAATATAAGAATAGAAGTTCGAATACCGCTGACGTCAC
TTTAATAAAAAATTGAGATTGACATGTAGCGGTCAATAAATATAGTATTAATATACAGCT
GTTCCAGTGGGTATGGCTGAATAAGACTTAATCTAACATGAAGATGAAGCAAAGGCCTCT
CCCATGTGTGGATATAGTNCGGCTACGCATCTCGGCAAAAAGTAATTATAACTATAAAAA
TCATGAAAAAGCCTCTAATTACCCGTATTCTGCCGAGTCATAGATAAAACGAAAATCGTG
TATACAGATTGGCCTCTGGGTAGATTAGAAGTGACATAAACTGAAACCAAGAACCAAAGG
TCTACGACCTCGCTGAGAACCAACTCTGTGTATGCAATTATGGATATTGTCATCATGTAA
TATATATCTAAATTATTTCTACTACAAATTATTTAGAAATAATTTGTAATAAAAATTATT
TAGAAATAAATTGTAATAAGAATTGTTTAGAGGTAATTTTTTAAAAATTCTTTAAAAATA
TTTTGTAATAAAAATTATTTAGAAATATTTTTNGAGGTGAGTTTCCGGGTCCAAATTTCT
AGGTNAATACCCGGAAACTCTCCCCGGGGCAGTTTTCGGTAACGAAATAAAATATAAGTG
TACGATTTTAATAAAAAGGATTGGTTCTCTAACAGGTTAACCCACTTTTAAAAATAAAAA
TCGCCTTTAGGCAGAGATGTGACCATTTTTCTTATTACTCATAAGATATNTAGGGTCTTA
GGCTTTCCCACCATTAAAAAGGTGGTTTCCTCCCATAAAAAAATCAATTTTTGATTATTT
AACTTATATTATAAATTTATTTATTAAAATAATTAGATAAAAAATCAAAATTCAGTCTTG
GGTTTTTTCACCCATGGGGGAAAAAACCCAAGATCCGATTTTCATGAAAAATTACTTCTA
TATCATGTATTTTAAATTTTTTAACAATTAATAGGTATTTACTAAATTTAACGTTGCGCA
CCAAATTTATAAATACCTATTAATTGTTTAAAAAAATTTAAATACATGATATAGAAGTAA
CTTTTCATGAAGATATTATTTTAA
```
*Figure 6. ORFs that was found in rnd-1_family-6#Unknown*
<img width="1220" alt="Снимок экрана 2025-02-17 в 20 47 14" src="https://github.com/user-attachments/assets/bc1a5a7b-b521-480f-b251-cd6feaf4c3c4" />

## **Conclusion**

The phylogenetic analysis identified several clusters of repeats, with clusters 7 and 92 standing out due to their substantial contributions to the D. muelleri genome. Cluster 92 occupies over 15% of the genome, while cluster 7 contributes more than 4%. Given the significant genomic impact of these clusters, they represent potentially interesting repeat elements that warrant further investigation. Further analysis may provide valuable insights into their functions and roles in genome organization and evolution.

